doc/includes/reveal/examples_reveal_modal_examples.html

```diff
@@ -10,13 +10,13 @@ <h2 id="firstModalTitle">This is a modal.</h2>
   <p>Reveal makes these very easy to summon and dismiss. The close button is simply an anchor with a unicode character icon and a class of <code>close-reveal-modal</code>. Clicking anywhere outside the modal will also dismiss it.</p>
   <p>Finally, if your modal summons another Reveal modal, the plugin will handle that for you gracefully.</p>
   <p><a href="#" data-reveal-id="secondModal" class="secondary button">Second Modal...</a></p>
-  <a class="close-reveal-modal" aria-label="Close">&#215;</a>
+  <a class="close-reveal-modal" aria-label="Close" role="button" tabindex="0">&#215;</a>
```

Comment:
This should be a <button> tag. aria-haspopup and role="button" not needed. `tabindex="0"` not needed because anchor tags are already in the tab order.